libs/community/langchain_community/embeddings/huggingface.py

```diff
@@ -154,6 +155,8 @@ class HuggingFaceInstructEmbeddings(BaseModel, Embeddings):
     """Instruction to use for embedding documents."""
     query_instruction: str = DEFAULT_QUERY_INSTRUCTION
     """Instruction to use for embedding query."""
+    show_progress: bool = False
```

Comment:
This used to default on and now defaults to off, not sure if you'd consider that a breaking change since it's just showing output.